tests/kafkatest/tests/client/client_compatibility_produce_consume_test.py

```diff
@@ -73,6 +75,7 @@ def min_cluster_size(self):
     @parametrize(broker_version=str(LATEST_2_8))
     @parametrize(broker_version=str(LATEST_3_0))
     @parametrize(broker_version=str(LATEST_3_1))
+    @parametrize(broker_version=str(LATEST_3_2))
     def test_produce_consume(self, broker_version, metadata_quorum=quorum.zk):
```

Comment:
I wonder if it's useful to continue testing older versions that we have not touched in years. Could we draw the line at 2.4 for example? For older versions, the main thing that seems useful to continue testing is the upgrade/downgrade tests. We don't necessarily have to change this in this patch of course, I'm just curious to hear what people think. cc @ijuma

Reply:
@hachikuji I agree with you. Kafka's system test are getting longer and longer to run. Should we move this discussion the mailing list or Jira to get some agreement here? cc @ijuma

Reply:
We also need to test client protocol compatibility with all the versions. I submitted KIP-896 recently to fix this in a holistic way.